frontend/server/src/Controllers/User.php

```diff
@@ -3700,22 +3741,32 @@ public static function getCoderOfTheMonthDetailsForSmarty(
     }
 
     /**
-     * @return array{smartyProperties: array{STATUS_ERROR: string}|array{profile: UserProfileInfo}, entrypoint: string}
+     * @return array{entrypoint: string, smartyProperties: array{payload: UserProfileDetailsPayload, title: \OmegaUp\TranslationString}}
      *
-     * @omegaup-request-param mixed $auth_token
-     * @omegaup-request-param string $contest_alias
-     * @omegaup-request-param null|string $token
      * @omegaup-request-param mixed $username
```

Comment:
```suggestion
     * @omegaup-request-param null|string $username
```

puedes cambiar esto en todas las funciones que mandan llamar `self::resolveTargetIdentity()`?

Reply:
Listo.